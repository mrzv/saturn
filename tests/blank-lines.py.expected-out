>>> a = 5                                                                       
... a                                                                           

5

