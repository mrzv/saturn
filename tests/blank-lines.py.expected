a = 5
a



#o> 5
